### ISSUE TITLE
Incorrect timezone handling in feed community blog post aggregator

### ISSUE THREAD

**Issue by timgraham** [django/djangoproject.com#241]
see https://code.djangoproject.com/ticket/23566 for details

**Comment by jezdez** [comment]
Yeah, this seems to be a problem with us not using the timezone support and assuming the default timezone of "America/Chicago". We should either turn the site into a UTC only site, or just enable timezone support. Any objects with either of those options?

**Comment by aaugustin** [comment]
I hear Django got fancy timezones support a few releases ago :D

**Comment by aaugustin** [comment]
(Just kidding -- if the path of least resistance is UTC, that's just fine.)

**Comment by jezdez** [comment]
@aaugustin Heh, I'm just nervous about all that old code not being prepared for timezones. I'll try to make it work with them.

**Comment by timgraham** [comment]
stalebot, please don't close.

**Comment by timgraham** [comment]
stalebot, please don't close.
